docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,229 @@
+
+It is recommended to run Envoy xDS management server (`clutch.module.chaos.experimentation.xds`) on a separate host.    
+
+### Configuration
+
+#### Frontend
+
+The frontend of the framework is completely configurable. Below is a sample [frontend config](https://github.com/lyft/clutch/blob/6990b5aa8b1e6a47a33b28e2aaab9783e4e9d084/frontend/packages/app/src/clutch.config.js) which will show the list of experiments and as well as workflow to start/stop an experiment. 
+
+```"@clutch-sh/experimentation": {
+module.exports = {
+  ...
+  "@clutch-sh/experimentation": {
+    listExperiments: {
+      description: "Manage fault injection experiments.",
+      trending: true,
+      componentProps: {
+        columns: [
+          { id: "target", header: "Target" },
+          { id: "fault_types", header: "Faults" },
+          { id: "start_time", header: "Start Time", sortable: true },
+          { id: "end_time", header: "End Time", sortable: true },
+          { id: "run_creation_time", header: "Creation Time", sortable: true },
+          { id: "status", header: "Status" },
+        ],
+        links: [
+          {
+            displayName: "Start Server Experiment",
+            path: "/server-experimentation/start",
+          },
+        ],
+      },
+    },
+    viewExperimentRun: {},
+  },
+  "@clutch-sh/server-experimentation": {
+    startExperiment: {
+      componentProps: {
+        upstreamClusterTypeSelectionEnabled: true,
+      },
+      hideNav: true,
+    },
+  },
+```
+
+#### Backend Server
+
+Below configuration will spin up all the required modules and services to store the data coming from frontend into Postgres database.
+
+```yaml title="backend/clutch-config.yaml"
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.api
+  - name: clutch.module.chaos.lyftexperimentation
+  - name: clutch.module.chaos.serverexperimentation
+services:
+  ...
+  - name: clutch.service.db.postgres
+      typed_config:
+        "@type": types.google.com/clutch.config.service.db.postgres.v1.Config
+        connection:
+          host: ${RDS_HOST}
+          port: ${RDS_PORT}
+          user: ${RDS_USER}
+          ssl_mode: REQUIRE
+          dbname: ${RDS_NAME}
+          password: ${CREDENTIALS_CLUTCH_TO_RDS_PASSWORD}
+  - name: clutch.service.chaos.experimentation.store
+```
+
+#### xDS Management Server 
+
+Below is the configuration for spinning up xDS server. For details about the fields, take a look at the [xds config proto](https://github.com/lyft/clutch/blob/main/api/config/module/chaos/experimentation/xds/v1/xds.proto).
+
+```yaml title="backend/clutch-xds-config.yaml"
+...
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.xds
+    typed_config:
+      "@type": types.google.com/clutch.config.module.chaos.experimentation.xds.v1.Config
+      rtds_layer_name: <RTDS_LAYER_NAME>                     // "rtds_layer"
```

Comment:
we should unify these yamls: right now we do `${RDS_HOST}` in some places and `<RTDS_LAYER_NAME> // "rtds_layer"` in other places -  let's choose one and follow it everywhere. Ideally we should follow whatever Clutch's convention is.